### PR TITLE
Added Institute for Shipboard Education to domains

### PR DESCRIPTION
**Pull request by wileybaba**
ISE is the institution behind the Semester at Sea program

### CODE REVIEW

**Comment by philipto** [comment]
@wileybaba Could you please clarify it the educational program offers long-term (> 1 year long) courses? If yes, could you please share the URL of the course page with us? Thank you!

**Comment by philipto** [comment]
@wileybaba You might miss my previous comment. Please react.

**Comment by wileybaba** [comment]
Hey @philipto ...ISE offers semester long programs and it is possible to take multiple semesters.  

https://www.semesteratsea.org/

**Comment by philipto** [comment]
@wileybaba Thank you for the clarification! 
**TL;DR**: please email community-support@jetbrains.com and ask for better suitable licensing option (it is most likely free anyway).

The issue is that according to this repository rules, 
> Your pull request for adding a new email domain to the repository will be satisfied if all of the conditions below are met:

> The domain is used by an educational institution, which offers at least one long-term course (one year or longer), and the course is somehow related to IT (it is in computer science, software engineering, statistics, bioinformatics, etc.)

> The educational institution is a physical entity with student attendance and recognized as providing a learning curriculum for the educational system, or the institution is an accredited online educational organization providing their students with: (1) online courses with a curriculum at least one year long, (2) a dedicated email address which is provided to students only until their graduation.

An ability to take multiple semesters is not the same as "at least one long-term course (one year or longer)". 

Thus, please email community-support@jetbrains.com and ask for a licensing option which better suits your needs and do not contradict with this repository rules, located at the bottom of [this page](https://github.com/JetBrains/swot). 

I am sorry, I have to close the request without merging, because this organization is a training center, not a college, nor a university. Thank you for the understanding!
